src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyTryWithResourceAloneSingle.java

```diff
@@ -31,9 +31,6 @@ void test() throws IOException {
         try (BufferedReader br1 = new BufferedReader(null);
                 BufferedReader br2 = new BufferedReader(br1)) { ; }
         catch (IOException e) { ; }
-        try (BufferedReader br1 = new BufferedReader(null);
-                // violation below ''}' at column 64 should be alone on a line'
-                BufferedReader br2 = new BufferedReader(br1)) {} catch (IOException e) { ; }
```

Comment:
Test input code should not be removed. It should stay as is, and either be a violation or not.

Reply:
@rnveach done